pkg/demo/demo.go

```diff
@@ -85,12 +85,28 @@ func New(pgpwd string) (*Demo, error) {
 	return app, nil
 }
 
+func (app *Demo) Query(ctx context.Context, dataset string, rows, cols []string) (string, error) {
+	if dataset == "skinny" {
```

Comment:
should this name be hardcoded?

Reply:
We don't necessarily have to do it this way. There only has to be a mechanism to switch between the two types of queries, and this was easy.
(The 'skinny' query does the pivot from a known table, and the other queries (so far) are normal row and column selects.)

Reply:
maybe some kind of `pivot` parameter?